google-cloud-firestore/src/main/java/com/google/cloud/firestore/spi/v1/FirestoreRpc.java

```diff
@@ -64,6 +66,8 @@
   UnaryCallable<ListCollectionIdsRequest, ListCollectionIdsPagedResponse>
       listCollectionIdsPagedCallable();
 
+  FirestoreClient.PartitionQueryPagedResponse partitionQuery(PartitionQueryRequest request);
```

Comment:
Do you think we'll want to have a method that doesn't auto handle the pages for us? I could envision a scenario where some data processing tool would want to handle pagination itself.

Reply:
This is where my area of expertise ends. I assume this suggest exposing 

```
 public final UnaryCallable<PartitionQueryRequest, PartitionQueryResponse>
      partitionQueryCallable();
```

If so, we can do so, but we don't have have an API to expose the page token. For one of our Firebase Storage APIs, we added the pageToken to a type that is similar to QueryPartition. Should we do that here as well? It will make very little sense as a callback for `getPartitions()` though.

Reply:
Yeah it's definitely different from the rest of the api. Let's hold off on adding something like that right now until we know it'll be needed.